Moclobemide/Moclobemide-Report.md

```diff
@@ -0,0 +1,383 @@
+
+**Excretion**: Less than 1% of a dose is excreted unchanged via the kidneys.
+
+# 2 Methods
+
+
+## 2.1 Modeling Strategy
+The general workflow for building an adult PBPK model has been described by Kuepfer et al. ([Kuepfer 2016](#5-references)). Relevant information on the anthropometry (height, weight) was gathered from the respective clinical study, if reported. Information on physiological parameters (e.g. blood flows, organ volumes, hematocrit) in adults was gathered from the literature and has been incorporated in PK-Sim® as described previously ([Willmann 2007](#5-references)). The  applied activity and variability of plasma proteins and active processes that are integrated into PK-Sim® are described in the publicly available 'PK-Sim® Ontogeny Database Version 7.3' ([PK-Sim Ontogeny Database Version 7.3](#5-references)).
+
+In general, the following step-wise workflow was followed:
+
+1. Fit total hepatic CL as a placeholder using single dose i.v. (50 mg ([Raaflaub 1984](#5-references)) and 150 mg ([Schoerlin 1987](#5-references))) data with renal clearance fixed to 0.034 ml/min/kg as derived from [Schoerlin 1987](#5-references) to select the appropriate distribution (i.e. partition coefficient) model.
+2. Estimate the contribution of non-CYP2C19 mediated metabolism using data (p.o.) from CYP2C19 poor metabolizers (PM) ([Yu 2001](#5-references), [Gram 1995](#5-references)). This pathway is assumed to be mainly attributed to FMO (flavin-containing monooxygenase) – however, also other unspecific CL would be captured here. An Asian individual was used for simulations of data reported by [Yu 2001](#5-references).
+3. Use single dose data (i.v. and p.o.) to estimate Vmax and Km of CYP2C19.
+4. Predict concentrations after multiple oral dosing and compare to literature. Steady state levels were not predicted very well, and the model was refined by including time-dependent auto-inhibition to account for a change in CL over time.
+5. Predict single and multiple doses profiles (both i.v. and p.o.) with the updated model and compare to published profiles. Qualify model by comparing predicted CL/F and Cmax to the corresponding parameters in a review across multiple studies. Population prediction to verify the variability components of the model.
+
+A typical European male subject (age = 30 years, weight = 73 kg, height = 176 cm, BMI = 23.57 kg/m2) was created in PKsim using the predefined database “European (ICRP, 2002)”, by adding CYP2C19 ( PK-Sim RT PCR database) and FMO (other) expression and used in simulations, until stated otherwise. For simulations of Asian subjects, a typical Asian individual (Age = 30 y, weight = 60.03 kg, height = 169.96 cm, BMI = 20.78 kg/m2) was created from the predefined database “Asian (Tanaka, 1996)” by adding CYP2C19 ( PK-Sim RT PCR database) and FMO (other) expression.
```

Comment:
_by adding CYP2C19 ( PK-Sim RT PCR database) and FMO (other) expression_
what does "other" mean here?

Reply:
Issue created